code/espurna/telnet.ino

```diff
@@ -42,6 +42,58 @@ void _telnetWebSocketOnConnected(JsonObject& root) {
 
 #endif
 
+#if TELNET_REVERSE_SUPPORT
+    void _telnetReverse(const char * host, uint16_t port) {
+        DEBUG_MSG_P(PSTR("[TELNET] Connecting to reverse telnet on %s:%d\n"), host, port);
+
+        unsigned char i;
+        for (i = 0; i < TELNET_MAX_CLIENTS; i++) {
+            if (!_telnetClients[i] || !_telnetClients[i]->connected()) {
+                #if TELNET_SERVER == TELNET_SERVER_WIFISERVER
+                    _telnetClients[i] = std::unique_ptr<WiFiClient>(new WiFiClient());
```

Comment:
```suggestion
                    _telnetClients[i] = std::make_unique<WiFiClient>();
```

Reply:
Fixed in https://github.com/xoseperez/espurna/pull/1920/commits/d62ecf2b6d8bfa814db8373255d4730699c9845e